### ISSUE TITLE
Crash during trying to generate a test with `: notnull`

### ISSUE THREAD

**Issue by CoenraadS** [mattwhitfield/Unitverse#115]
**Describe the bug**
Crash during trying to generate a test with `: notnull`

**To Reproduce**
```
public class Crash<TRequest> where TRequest : notnull
{
    ///
    public Crash(TRequest foo)
    {
    }
}
```

Try to make a test from this class will popup message box with exception

**Comment by mattwhitfield** [comment]
Turns out this wasn't so hard - build incoming

**Comment by mattwhitfield** [comment]
Resolved in 0.131 - thanks again 👌
